Test dev

pgleeson opened the pull request.

mwatts15 committed:
Specifying branch for owm clone

- Temporary until c302 is not using the project directly

mwatts15 committed:
Dropping dev part from version

mwatts15 committed:
Bumping version number

mwatts15 committed:
Updating for changes to owmeta_core.identifier_mixin

mwatts15 committed:
Allowing dev version for owmeta-core

mwatts15 committed:
Specifying value type for DWEDS contexts

- Prevents ambiguity since the combined context has the same ID as the DWEDS

mwatts15 committed:
Specifying value type for DWEDS contexts

- Prevents ambiguity since the combined context has the same ID as the DWEDS

mwatts15 committed:
Adding owmeta-pytest-plugin as a test requirement

mwatts15 committed:
Adding owmeta-pytest-plugin as a test requirement

mwatts15 committed:
WIP: Added SourcedFrom property

mwatts15 committed:
Updating for rename of DataSource.commit/commit_augment

mwatts15 committed:
Updating for rename of DataSource.commit/commit_augment

mwatts15 committed:
wip: adding support for evidence ds mixin

mwatts15 committed:
Updating for changes for embedded transformations in core

mwatts15 committed:
Adding an install script for CI

mwatts15 committed:
Adding Github workflows

mwatts15 committed:
Removing defunct setup configs

- Also adding -x, -e to ci-install.sh

mwatts15 committed:
Correcting dev branch reference in CI

mwatts15 committed:
Adjusting for rdflib.Graph.serialize interface change in 6.0

- Caused by rdflib/rdflib#1183

mwatts15 committed:
Adding PubMed API key secret

mwatts15 committed:
Adding a constraint for pyparsing ver on py 3.6

- Also adding 3.9 and 3.10 classifiers to project metadata

mwatts15 committed:
Fixing version constraint specification

mwatts15 committed:
Adding a wrapper for Git requests

mwatts15 committed:
Adding automatic retries to document elaboration HTTP requests

mwatts15 committed:
Allowing to pass in requests arguments for doc elaboration

mwatts15 committed:
Removing RequestsResponseStream

mwatts15 committed:
Raising for doc elaboration HTTP request errors

mwatts15 committed:
Updating a couple of links in the readme

mwatts15 committed:
Merge remote-tracking branch 'origin/dev' into evidence-ds-mixin

mwatts15 committed:
Adjusting for github_token suddenly being reserved

mwatts15 committed:
Updating badge

mwatts15 committed:
Adding a test for SourcedFrom

mwatts15 committed:
Merge remote-tracking branch 'origin/dev' into evidence-ds-mixin

mwatts15 committed:
Adding automated deploy of dev snapshots from CI

mwatts15 committed:
Disabling wormbase document elaboration tests in CI temporarily

mwatts15 committed:
Making Document elaboration response handling more robust

- Should not fail for lack of content-type in response since we
  only retrieve it for determining the character encoding
- We do expect a successful response, so fail first if we do not
  get one

mwatts15 committed:
Attempting to correct deploy.sh for owmeta

mwatts15 committed:
Attempting (again) to correct deploy.sh for owmeta

mwatts15 committed:
Re-enabling wormbase tests

mwatts15 committed:
Adding some detail on how DWEDS IDs work

mwatts15 committed:
Reducing scheduled dev builds to 4 per day

mwatts15 committed:
Setting class registry context for test configs

mwatts15 committed:
Switching to the official requests sphinx ref

- See https://github.com/psf/requests/issues/6140#issuecomment-1135071992 for details

mwatts15 committed:
Adding transaction managers for docs/examples

mwatts15 commented:
@pgleeson not sure what conflicts GitHub sees: I tried merging locally and Git just "fast-forwarded".

mwatts15 committed:
fixing a couple of tests

mwatts15 committed:
Correcting context for data context declaration in README

- The supports query constrains to contexts, but the evidence context lacked the
  rdf:type statement for the data context

pgleeson committed:
Update README.md

pgleeson committed:
Update scheduled-dev-build.yml

pgleeson committed:
Update scheduled-master-build.yml
